conference/homepage.py

```diff
@@ -1,5 +1,42 @@
 from django.template.response import TemplateResponse
+from django.utils.lorem_ipsum import words, paragraphs
+from django import forms
 
 
 def homepage(request):
-    return TemplateResponse(request, 'ep19/homepage.html', {})
+    return TemplateResponse(
+        request, 'ep19/bs/homepage/home.html', {}
+    )
+
+
+def generic_content_page(request):
+    return TemplateResponse(
+        request, 'ep19/bs/content/generic_content_page.html', {
+            'lorem_words': words,
+            'lorem_paragraphs': paragraphs,
+        }
+    )
+
+
+def generic_content_page_with_sidebar(request):
+    return TemplateResponse(
+        request, 'ep19/bs/content/generic_content_page_with_sidebar.html', {
```

Comment:
Might need to fix the header :) 

<img width="948" alt="screen shot 2019-03-05 at 7 47 13 pm" src="https://user-images.githubusercontent.com/667029/53854740-8c350100-3fc2-11e9-8a41-63e5258c827a.png">

Reply:
I think that's because the title comes from django cms.